### ISSUE TITLE
Failed to show texture in new seabed

### ISSUE THREAD

**Issue by arainbilal** [uuvsimulator/uuv_simulator#340]
Following the tutorial "making a simple seabed" (https://uuvsimulator.github.io/tutorials/seabed_world.html), the texture is not appearing properly. I think there is more to do in the Blender when exporting the mesh to Collada DAE file for the Gazebo world. Indeed, it can be seen in the tutorial page that the sand.material is used for a texture but when its imported in Gazebo, it is no where close to the the texture file given in sand.material. I think there are two things to be considered: 1) If the mesh grid is too small, the texture won't appear properly. The fine grid is due to the interpolation applied in the python script (see tutorial page); 2) The material scaling should be applied properly.

To this end, I have performed multiple tests in using different settings but cannot get the texture applied to the tutorial mesh (above weblink) similar to the world_models/height_sandmap. I request to please include few more steps or provide some guidance in the tutorial page or in repose to this issue about how the texture can appropriately can be applied on the tutorial seabed mesh created using Python scripts to be visualize appropriately in the Gazebo. 

I think that the answer to this issue will be helpful to all other who have less experience with Blender and are trying to create world files with appropriate texture visualizing the uuv_simulator environment.

**Comment by musamarcusso** [comment]
Hi @arainbilal, thanks for the heads up. I will check the documentation. Blender is indeed quite a challenge.

**Comment by arainbilal** [comment]
Hi @musamarcusso . Thanks. I have now attached the gazebo world images showing the tutorial seabed and the sand_heightmap, which may help to elaborate the issue. In both files, I have used the sames material in the sdf file as follows:
`<uri>file://tutorial_seabed_world/materials/scripts/sand.material</uri>`
The objective is to display the sand material on the tutorial seabed, similar to the one showing in attached sand_heightmap. Hope to get more info soon.

Tutorial seabed with sand.material
![tutorialseabed](https://user-images.githubusercontent.com/9365220/53624992-a9c63b80-3c4d-11e9-8bf2-77a138cb3a20.png)

sand_heightmap with sand.material
![sand_heightmap](https://user-images.githubusercontent.com/9365220/53624999-ac289580-3c4d-11e9-9df1-8aec64cffada.png)
